packages/menu/src/styles.tsx

```diff
@@ -62,6 +62,16 @@ export const menuItemContainerStyle = css`
   }
 `;
 
+export const menuItemHeight: Record<Size, string> = {
+  [Size.Default]: css`
+    min-height: 34px;
+  `,
+
+  [Size.Large]: css`
+    min-height: 46px;
+  `,
+};
```

Comment:
Since these are height properties, it feels like these should be multiples of 8px? If you set it to these values directly from Figma, maybe it's worth confirming with design.

Reply:
Default is 32 + 2x2 padding, Large is 32 + 2x7 padding. I'll double check the 7px 👍